src/main/java/org/terasology/potions/PotionStatusWidget.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.potions;
+
+import org.terasology.math.geom.Vector2i;
+import org.terasology.rendering.nui.Canvas;
+import org.terasology.rendering.nui.CoreWidget;
+import org.terasology.rendering.nui.UIWidget;
+import org.terasology.rendering.nui.databinding.Binding;
+import org.terasology.rendering.nui.databinding.ReadOnlyBinding;
+import org.terasology.rendering.nui.layouts.ColumnLayout;
+import org.terasology.rendering.nui.widgets.UILabel;
+
+public class PotionStatusWidget extends CoreWidget {
+
+    private ColumnLayout effects = new ColumnLayout();
+
+    public PotionStatusWidget() {
+        effects.setFillVerticalSpace(false);
+        effects.addWidget(new UILabel("Potion Effects:"));
```

Comment:
This will always show the label, even if no potions are active. This could be improved later on.